github_get_env.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -ux
+env
+set
+uname -a
+sw_vers
+xcodebuild -version
+system_profiler -timeout 30 -detailLevel basic
+sudo launchctl list
+launchctl list
+sudo sysctl -a
+sudo kextstat
+df -h && ls -kahl
+sudo mdutil -a -i off # disable spotlight
+sudo mdutil -d /
```

Comment:
Nitpick: would be better to add an empty line to EOF.

Reply:
I was wondering why to add a `newline` to the last line and found [this of stackoverflow](https://stackoverflow.com/a/729795) referring to the [POSIX.1-2017 definitions of a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) – if anyone else is wondering too.
Will change the files to include a final `newline` character